snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -83,6 +83,14 @@ init_config:
   #     metric_tags:
   #       - TCP
 
+  ## @param profiles - dictionary - optional
+  ## Specify profiles to be able to reference a set of metrics by name.
+  ## It points to a file with the same structure as global_metrics.
+  #
+  # profiles:
+  #   profile1:
+  #   	definition: profile1.yaml
```

Comment:
Any reason to not just include them in this file?

Reply:
And wouldn't it be more convenient from user perspective if we (also?) provide a way to inline the metrics directly in the configuration file (like `jmx`) ?

```yaml
profiles:
    profile1:
        - OID: "1.3.6.1.2.1.7.1.0"
           name: "IAmACounter32"
        - OID: "1.3.6.1.2.1.7.1.0"
           name: "IAmACounter32"
        - OID: "1.3.6.1.2.1.7.1.0"
           'name': "IAmACounter32"
```

Reply:
I added the ability to inline it.